clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -317,6 +328,14 @@ Timer timer() {
             return timer;
         }
 
+        void setEnqueuedMs(final long enqueuedMs) {
```

Comment:
could add a java doc for this, the getter and var. Mainly to clarify its the time when it was added to the queue (not a duration/time it's been on the queue, which is another concept we are working with, that's why I think it's worth clarifying)....we could also consider `enqueueTimeMs` as name.